src/pyocr/libtesseract/__init__.py

```diff
@@ -212,11 +212,13 @@ def is_available():
     if not available:
         return False
     version = get_version()
+
     # C-API with Tesseract <= 3.02 segfaults sometimes
     # (seen with Debian stable + Paperwork)
     # not tested with 3.03
     if (version[0] < 3 or
             (version[0] == 3 and version[1] < 4)):
+        print("Unsupported version [%s]" % ".".join([str(r) for r in version]))
```

Comment:
Please use the Python logging infrastructure:
```py
import logging
logger = logging.getLogger(__name__)
(...)
    logger.warning("Libtesseract: Unsupported version [%s] ...")
```

Reply:
Sure - Can do. Do you want me to add any debug logging while I'm at it?

Reply:
It's good enough for this PR, thanks :)